Sources/DNA/Font/Font.swift

```diff
@@ -34,66 +37,78 @@ public extension UIFont {
         registerCustomFonts()
 
         let font = UIFont(name: FontType.light.rawValue, size: 28.0)!
-        return font.scaledFont(forTextStyle: .title2)
+        return font.scaledFont(forTextStyle: .title1)
     }
 
-    /// FINNTypeWebStrippet-Light with a size of 22.5 scaled for UIFontTextStyle.title3
+    /// FINNTypeWebStrippet-Light with a size of 22 scaled for UIFontTextStyle.title3
     ///
     /// ## Usage:
     /// - If a T2 have more sublevels, you can use T3.
     /// - This always has the weight Light.
     public static var title3: UIFont {
         registerCustomFonts()
 
-        let font = UIFont(name: FontType.light.rawValue, size: 22.5)!
-        return font.scaledFont(forTextStyle: .title3)
+        let font = UIFont(name: FontType.light.rawValue, size: 22)!
+        return font.scaledFont(forTextStyle: .title2)
```

Comment:
Same here, shouldn't this be title3?

Reply:
Because in Human Interface guidelines their Title2 is the same side as our title3, so just to her the dynamic type scaling to work in the way they have put it

Reply:
Ahhh, I see!